### PR TITLE
Refactor lodash modules to cherry-picked usage

### LINKED ISSUE #4086: Replacing deprecated lodash modules
Hi Dreamteam!

I noticed Expo is using the modular packaged from Lodash. [At CommitLint someone noticed us](https://github.com/conventional-changelog/commitlint/pull/473) that these [packages are deprecated](https://github.com/lodash/lodash/issues/3838#issuecomment-398592530) and not being updated anymore. The [roadmap also states it will drop support](https://github.com/lodash/lodash/wiki/Roadmap#v500-2018) for these modules.

I would recommend swapping to the main build for security and support reasons. Luckily there are still ways to import specific functions from Lodash. You could either tree-shake it, and if that isn't available you can "cherry pick" it:

1. `import { zipObject } from 'lodash';`
2. `const zipOjbect = require('lodash/zipObject');`

I'd be happy to help with this if you agree! 😄 No hard feelings if you don't ❤️ 

Cheers,
❤️ Cedric

### PR DESCRIPTION
**Pull request by byCedric**
# Why

This PR fixes #4086, Lodash has dropped support for the separate module packages. With this, you are using an "active" supported Lodash version.

# How

I basically replaced all usages of the `lodash.<package>` with `lodash` in the `package.json` and updated their imports to `require('lodash/<package>')`.

# Test Plan

It's a relatively small change, and if I screwed up the CI should show my (character) flaws.

> ~~Note, I'm not sure if I correctly rebuilt the yarn lock file. It also adds some extra dependencies, it always happens to me and yarn, but I guess it's just me...~~ Seems to be working fine after a reinstall of yarn on my machine

### CODE REVIEW

**Comment by byCedric** [comment]
Let me know if I need to merge/rebase/squash or abuse my messages in any way, happy to do so!

File: packages/expo-sqlite/build/SQLite.js [hunk 1]
```diff
@@ -56,7 +56,9 @@ function _escapeBlob(data) {
 }
 const _openExpoSQLiteDatabase = customOpenDatabase(SQLiteDatabase);
 function addExecMethod(db) {
-    db.exec = db._db.exec;
+    db.exec = (queries, readOnly, callback) => {
```
Comment (byCedric) [inline packages/expo-sqlite/build/SQLite.js]: I swear I didn't add this, it must be my typescript compiler

Comment (sjchmiela) [reply packages/expo-sqlite/build/SQLite.js]: Or a not up-to-date JS 🙃

**Comment by byCedric** [comment]
Ok, so I can remove my "builds". I thought it would save you guys some time, but I feel like unexplainable things (like extra methods, typing etc) is added. Might be the latest master, might be #me. Just say the word and I'll revert those builds, keeping the ts/package changes

File: yarn.lock [hunk 2]
```diff
@@ -13608,6 +13574,62 @@ react-native-webview@5.4.6:
     escape-string-regexp "1.0.5"
     invariant "2.2.4"
 
+react-native@0.59.5:
```
Comment (ide) [inline yarn.lock]: This doesn't seem right - you maybe need to check out the submodules before running Yarn.

Comment (byCedric) [reply yarn.lock]: That's what I thought too, I'll clean my system and retry it tonight. Thanks for the tip! ❤️

Comment (byCedric) [reply yarn.lock]: Turned out some unexplainable yarn config from another project to make it run. Cleaned it, removed it and now it looks way better!

**Pull request merged**